rs3-full-boss-guides/magister.txt

```diff
@@ -1,178 +1,310 @@
-      },
-      {
-        "name": "__100k HP__",
-        "value": "⬥ Another soul obelisk spawns to the south, which deals 600 damage and raises the player's corruption level by 4% every 5-7 seconds. [Example](https://imgur.com/D3nIbDG.png)\n\u00a0\u00a0\u00a0\u00a0• Corruption causes the player's damage dealt and taken to increase by the same percentage.\n\u00a0\u00a0\u00a0\u00a0• Caps at 50%, and resets down to 25% after every kill.\n\u200B"
-      },
-      {
-        "name": "__50k HP__",
-        "value": "⬥ The Magister goes berserk:\n\u00a0\u00a0\u00a0\u00a0• Increases his damage output while losing some defence.\n\u200B"
-      },
-      {
-        "name": "__Resonance__",
-        "value": "⬥ Magister will heal for the damage you would have dealt. [Example](https://imgur.com/HcUrNlA.mp4)\n\u00a0\u00a0\u00a0\u00a0• It is best to use a bleed or a weak ability to minimise his healing.\n\u200B"
-      },
-      {
-        "name": "__Freedom__",
-        "value": "⬥ Magister cannot freedom while stunned, he will only freedom when he is bound. Should you bind him before or during a bleed, he can clear it. [Example](https://imgur.com/GkerL8m.mp4)\n\u200B"
-      },
-      {
-        "name": "__Siphon__",
-        "value": "⬥ When the player is at 70%+ adrenaline, the Magister can drain some of their adrenaline. [Example](https://imgur.com/8h2rcWI.mp4)\n\u00a0\u00a0\u00a0\u00a0• Aim to stay below 70% adrenaline during ultimates.\n\u00a0\u00a0\u00a0\u00a0• Be wary of adrenaline drain prior to ultimates.\n\n*Note: The three abilities mentioned above replace an auto attack and instead delay Magister's next auto attack by only 0.6s.*\n\u200B"
-      }
-    ]
-  }
-}
-.embed:json
+__**Action: Freedom**__
+.img:https://i.imgur.com/GkerL8m.gif
+⬥ Magister will use Freedom <:freedom:535541258240786434>, clearing any binds and bleeds on him.
+⬥ He will only use this if he is bound - e.g. (TighT) Binding Shot <:bindingshot:535541306563231790> <:tight:535541275957657600>, Ice Barrage <:icebarrage:537340400185245701>, etc.
+⬥ Simply ignore it and move on.
+
+.
+__**Action: Siphon**__
+.img:https://i.imgur.com/8h2rcWI.gif
+⬥ Magister will use Siphon <:Siphon:553050198012854275> draining 10% of the player's adrenaline.
+⬥ Simply ignore it and move on.
+
+.
+__**Action: Resonance**__
+.img:https://i.imgur.com/HcUrNlA.gif
+⬥ Magister use Resonance <:res:535541258844635148> and heal off the next hit the player does.
+⬥ Simply ignore it and move on.
+
+.
+__**Action: Restore Me**__
+.img:https://i.imgur.com/3siuSRJ.gif
+⬥ Magister will shout `Restore Me!` and heal 6-12k health.
+⬥ Simply ignore it and move on.
```

Comment:
Perhaps note that stunning will cancel it (if it still does, haven't been to the boss in forever). If not just ignore this comment